presto-orc/src/main/java/com/facebook/presto/orc/TupleDomainFilter.java

```diff
@@ -69,6 +70,11 @@
 
     boolean testBoolean(boolean value);
 
+    default boolean testBoolean(byte value)
```

Comment:
conventions: Don't add default implementation. Just add implementation in all places like other methods.